frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -1219,6 +1252,30 @@ export function useMarkerGenes({
   );
 }
 
+export function useGeneInfo(geneSymbol: string): UseQueryResult<GeneInfo> {
+  const { data } = usePrimaryFilterDimensions();
+  const genesByName = useMemo((): { [name: string]: OntologyTerm } => {
+    let result: { [name: string]: OntologyTerm } = {};
+
+    if (!data) return result;
+
+    const { genes } = data;
+
+    result = generateTermsByKey(genes, "name");
+
+    return result;
+  }, [data]);
+
+  const geneID = genesByName[geneSymbol]?.id;
+  return useQuery(
+    [USE_GENE_INFO, geneSymbol, geneID],
+    ({ signal }) => fetchGeneInfo({ geneSymbol, geneID, signal }),
```

Comment:
https://github.com/chanzuckerberg/single-cell-data-portal/blob/6ecc8f64c80287d28da853dabb373a7a6ae46830/backend/gene_info/api/ncbi_provider.py#L49

>>> Given a gene ensembl ID and gene name, returns a tuple with NCBI's corresponding gene UID and a boolean noting if the result is from searching by the gene name, instead of the gene ENSEMBL id. Initially, uses ensembl ID to find gene UID, but in the event that this returns an unexpected result, call the NCBI API again with gene name. This is successful if the response returns only 1 result for UID.

TL;DR - sometimes using the UUID doesn't work if it's out-of-date and so the gene symbol is a fallback.